static/module/search.js

```diff
@@ -73,4 +73,14 @@ export class Search {
       score,
     }))
   }
+
+  // return all indexed packages using minisearch's wildcard query
+  all() {
+    this.stringSearch = false
+    const wildcard = this.minisearch.constructor?.wildcard
+    if (!wildcard) {
+      throw new Error('minisearch wildcard symbol is unavailable')
```

Comment:
LOL, of great help of course.  For the end user!  🏩

Reply:
where is a `sublime.status_message()` if you need one ;)